src/Media/Dto/UploadDataDto.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Sher\Sdk\Media\Dto;
+
+use JMS\Serializer\Annotation as Serializer;
+use Sher\Sdk\Media\Enum\TypeEnum;
```

Comment:
```suggestion
use Sherl\Sdk\Media\Enum\TypeEnum;
```
Il manque un "l" à Sherl dans l'import.